listenbrainz/webserver/views/playlist_api.py

```diff
@@ -75,9 +75,16 @@ def validate_playlist(jspf):
         return
 
     for i, track in enumerate(jspf["playlist"].get("track", [])):
-        recording_uri = track.get("identifier")
-        if not recording_uri:
-            log_raise_400("JSPF playlist track %d must contain an identifier element with recording MBID." % i)
+        recording_uris = track.get("identifier")
+        if not recording_uris:
+            log_raise_400("JSPF playlist track %d must contain an identifier list with at least one recording MBID." % i)
+
+        # This allows identifier to be a list, tuple or string. The string support is a leftover and should be removed
+        # after 2025-06, which marks a year or backward compatibility. Only the first option of the if statement should remain.
+        if isinstance(recording_uris, list) or isinstance(recording_uris, tuple):
```

Comment:
This should be removed as well since we are only doing 1 identifier?

Reply:
yep